### DISCUSSION TITLE
Type error within `umzug` > `pony-cause`

### DISCUSSION THREAD

**Discussion by SpencerKaiser** [mikro-orm/mikro-orm#3386] [General]
Not sure where this is most relevant, but I just started [a discussion on the `umzug` repo](https://github.com/sequelize/umzug/discussions/561) about a type error with a dependency of theirs. If they end up updating the package to incorporate the new fix, can we get a patch to bump `umzug` here too?

**Comment by SpencerKaiser** [comment]
@B4nan [FYI `umzug` was just updated to 3.1.2 and it has the fix for `pony-cause`](https://github.com/sequelize/umzug/discussions/561#discussioncomment-3386355)

**Reply by SpencerKaiser** [reply]
After posting I realized you aren't on `3.1.x`, but the maintainer over there said the patch was applied to `3.2.x` as well!

**Reply by B4nan** [reply]
Its there already in master

https://github.com/mikro-orm/mikro-orm/commit/909f77ba169ef08793f35f1518bc41bbfd413652

**Reply by SpencerKaiser** [reply]
@B4nan you're fast!!! Thanks brotha

**Reply by SpencerKaiser** [reply]
@B4nan it's in `master` but no release has been made, right? Think it'll get into `5.3.2`?

**Reply by B4nan** [reply]
Every commit to master is out as dev version.
